x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/collector/cluster/ClusterStatsMonitoringDocTests.java

```diff
@@ -351,9 +351,7 @@ public void testToXContent() throws IOException {
         when(mockJvmInfo.getBundledJdk()).thenReturn(true);
         when(mockJvmInfo.getUsingBundledJdk()).thenReturn(true);
 
-        final Build mockBuild = mock(Build.class);
-        when(mockBuild.flavor()).thenReturn(Build.Flavor.DEFAULT);
-        when(mockBuild.type()).thenReturn(Build.Type.DOCKER);
+        final Build mockBuild = new Build(Build.Flavor.DEFAULT, Build.Type.DOCKER, "", "", false, "");
```

Comment:
Similar question about this test change.

Reply:
Same purpose here. We can't mock records and Mockito usually advice against mocking data classes.